.travis/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #  'RUN .travis/install-verilator.sh' to this dockerfile to have an image
 #  with the full environment already installed.
 #
-FROM ubuntu:trusty
+FROM librecores/librecores-ci:latest
 RUN apt-get update && apt-get install -y \
```

Comment:
Many tools below are bundled into the image (for FuseSoC). I suggest cleaning it up.
Also, is this file needed at all after switching to the new base image?

Reply:
Yes oleg , this file can be removed , though it works on local machine , I will send a commit removing it